salt/modules/debbuild.py

```diff
@@ -755,19 +778,23 @@ def make_repo(repodir,
                 number_retries = timeout / interval
                 times_looped = 0
                 error_msg = 'Failed to reprepro includedsc file {0}'.format(abs_file)
-                cmd = 'reprepro --ignore=wrongdistribution --component=main -Vb . includedsc {0} {1}'.format(codename, abs_file)
+                cmd = 'reprepro --ignore=wrongdistribution --component=main -Vb . includedsc {0} {1}'.format(
+                        codename,
+                        abs_file)
+
+#               env=gpg_tty_info_dict,
```

Comment:
Let's also please remove this commented line.

Reply:
done